csvparser/csvParser.go

```diff
@@ -179,3 +179,14 @@ func writeSQLForHierarchy(filePrefix string, h *sql.Hierarchy) {
 	sql.WriteSQL(file, h)
 	fmt.Printf("Finished writing %s with %d entries\n", filename, len(h.Entries))
 }
+
+func describeHierarchyEntries(entries map[string]bool) string {
+	if len(entries)>100 {
+		return fmt.Sprintf("containing %d entries", len(entries))
```

Comment:
Any particular reason why we only log this for entries that has more than 100 in them?